trustfall_core/src/schema/adapter/schema.graphql

```diff
@@ -18,6 +18,25 @@ type RootSchemaQuery {
     method for adapters over this schema.
     """
     Entrypoint: [Edge!]!
+    
+    Schema: Schema!
```

Comment:
didn't know what the comment should be here so I left it blank but we probably should add one

Reply:
How about: `Information about the schema itself.` or something along those lines?

Reply:
Sounds great